PWGHF/correlationHF/AliAnalysisTaskSELambdacCorrelations.cxx

```diff
@@ -114,6 +114,7 @@ AliAnalysisTaskSE(),
   fCentEvV0M(0.),
   fzVtx(0.),
   fMEAxisThresh(kFALSE),
+  fSoftPiCut(kTRUE),
```

Comment:
Better to set (here and in the following) the default as kFALSE, also for backward compatibility, considering that we still don't know whether we'll actually use this selection or not